meta/etcd.go

```diff
@@ -0,0 +1,100 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package meta
+
+import (
+	"context"
+	"time"
+
+	"cqfn.org/degitx/discovery"
+	"cqfn.org/degitx/misc"
+
+	ma "github.com/multiformats/go-multiaddr"
+	"go.etcd.io/etcd/clientv3"
+)
+
+type etcd struct {
+	Endpoints []string
+}
+
+func NewEtcd(endpoints ...string) (Storage, error) {
+	var urls []string
+	for _, endpoint := range endpoints {
+		multiaddr, err := ma.NewMultiaddr(endpoint)
+		if err != nil {
+			return nil, err
+		}
+		addr := new(discovery.MaNetworkAddr)
+		if err := addr.Parse(multiaddr); err != nil { //nolint:dupl // just parsing an address
+			return nil, err
+		}
+		urls = append(urls, addr.String())
+	}
+
+	return &etcd{
+			Endpoints: urls,
+		},
+		nil
+}
+
+const conTimeout = time.Second * 5
```

Comment:
@stepanvalyavskiy let's give a more explicit name to this `const`, since it'll be accessible across other files in the same package, e.g. `etcdConTimeout`